Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/subdisc/HL7PRPA201306Transforms.java

```diff
@@ -126,7 +130,7 @@ public PRPAIN201306UV02 createPRPA201306ForPatientNotFound(PRPAIN201305UV02 oReq
         boolean bRequiredFieldsAreNull = areIncommingRequiredPRPAIN201305FieldsNull(oRequest);
         if (bRequiredFieldsAreNull) {
             LOG.error("One or more required fields from the patient discovery request "
-                + "for the patient not found scenario are null.");
+                    + "for the patient not found scenario are null.");
```

Comment:
I agree, but I did not touch this code.  The eclipse formatter made the change since I saved the file.  So I think I should leave it alone for my commits for FHAC-760?